lib/internal/bootstrap/web/exposed-window-or-worker.js

```diff
@@ -59,27 +59,16 @@ installObjectURLMethods();
 
 {
   // https://fetch.spec.whatwg.org/#fetch-method
-  function set(value) {
-    ObjectDefineProperty(globalThis, 'fetch', {
-      __proto__: null,
-      writable: true,
-      value,
-    });
-  }
   ObjectDefineProperty(globalThis, 'fetch', {
     __proto__: null,
     configurable: true,
     enumerable: true,
-    set,
-    get() {
-      function fetch(input, init = undefined) {
-        // Loading undici alone lead to promises which breaks lots of tests so we
-        // have to load it really lazily for now.
-        const { fetch: impl } = require('internal/deps/undici/undici');
-        return impl(input, init);
-      }
-      set(fetch);
-      return fetch;
+    writable: true,
+    value: function fetch(input, init = undefined) { // eslint-disable-line func-name-matching
+      // Loading undici alone lead to promises which breaks lots of tests so we
+      // have to load it really lazily for now.
+      const { fetch: impl } = require('internal/deps/undici/undici');
```

Comment:
do we really need to lazily load it every time?
Can't we have something around the lines of:
```javascript
let impl;

value: function fetch(input, init = undefined) {
  impl ??= require('internal/deps/undici/undici');
  return impl(input, init)
},
```

Reply:
Technically require would cache it anyway, right?

Reply:
Well, technically it will, but the cost for calling require would still be higher (I don't have benchmarks, but it has more overhead than holding a local variable in memory)